index.html

```diff
@@ -39,11 +40,38 @@ <h1>Create a Quiz Question</h1>
       <button type="button" id="shuffle-options">Shuffle Options</button>
       <button type="submit">Save Question</button>
     </form>
-  </div>
-  <h2 class="section-heading">Quiz Questions</h2>
+  </section>
+
+  <section>
```

Comment:
Good idea to use the `<section>` tag instead of the `<div>` tag when you are using it for an actual new section of the page. 🧠